doc/check.texi

```diff
@@ -1301,11 +1302,12 @@ srunner_add_suite (sr, make_pack_suite ());
 @end verbatim
 @end example
 
-@node Selective Running of Tests, Testing Signal Handling and Exit Values, Multiple Suites in one SRunner, Advanced Features
+@node Selective Running of Tests, Selecting Tests Based on Arbitrary Tags, Multiple Suites in one SRunner, Advanced Features
```

Comment:
This may be too picky, but I feel that the section covering selectively running tests should cover both the idea of a suite or test case as well as tagging. That is, the following would be the sections:
```
4.6 Selective Running of Tests
4.6.1 Selecting Tests by Suite or Test Case
4.6.2 Selecting Tests by Tags
```

Let me give a stab at how this may flow:

```
4.6 Selective Running of Tests
After adding a couple of suites and some test cases in each, it is sometimes practical to 
be able to run only one suite, or one specific test case, without recompiling the test code. 
Check provides two ways to accomplish this, either by specifying a suite or test case by name
or by assigning tags to test cases and specifying one or more tags to run.

4.6.1 Selecting Tests by Suite or Test Case
There are two environment variables available that can be used to run a specific suite or
test case by name: CK_RUN_SUITE and CK_RUN_CASE. Just set the value to the name
of the suite and/or test case you want to run. These environment variables can also be a 
good integration tool for running specific tests from within another tool, e.g. an IDE.

4.6.2 Selecting Tests by Tags
(the content you wrote).
```